appliances/upgrading.md

```diff
@@ -31,6 +31,8 @@ In this example, we will update from jansa to kasparov.
         # reboot
 
 
+**Note:** If upgrading to Quinteros, messaging (Kafka) setup is required. See [here](https://github.com/ManageIQ/manageiq-documentation/blob/master/_includes/configuration.md#configure-messaging)
```

Comment:
Not sure if this is the correct way to link to another doc, let me know

Reply:
No, you wouldn't link to the includes this way.  It's hard to tell but first you need to include the partial in a full doc.  Then you would like to the full doc via an absolute path from the root.

I can help you out over voice if that helps.